tags/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Domain and page objects
+class Domain(models.Model):
+    name = models.CharField(max_length=100, default='', unique=True)
+    url = models.URLField(max_length=300, blank=False, null=False)
+
+class Page(models.Model):
+    url = models.URLField(max_length=300, blank=False, null=False)
+    domain = models.ForeignKey(Domain, on_delete=models.CASCADE)
+
+    #from eyehistory
+    title = models.CharField(max_length=2000, default='')
+    favicon_url = models.TextField(default='')
```

Comment:
It looks like both are being used o.O api/resources.py, line 287 and 306.

![image](https://cloud.githubusercontent.com/assets/20367063/24030059/eb25d962-0ab0-11e7-9271-4f9bd203e3e2.png)